compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -1887,6 +1887,19 @@ AttentionOp::getTiledImplementation(OpBuilder &builder,
   // Scale
   tiledOperands.emplace_back(scale);
 
+  // Mask
+  Value attnMask = getMask();
+  if (attnMask != nullptr) {
```

Comment:
```suggestion
  if (Value attnMask = getMask()) {
```

Reply:
This would pass even when attnMask is <<NULL_VALUE>>

Reply:
Same as below, this should work, you may be trying to work around some other bug.

Reply:
See below